test/object_test.go

```diff
@@ -345,27 +337,45 @@ func TestObjectMetadata(t *testing.T) {
 	_, err = obs.PutString("A", "AAA")
 	expectOk(t, err)
 
-	meta := &nats.ObjectMeta{Name: "B"}
+	meta := &nats.ObjectMeta{Name: "A"}
```

Comment:
Using "B" was on purpose.

Reply:
B is further down. I added meta data with update since the PutString doesn't take meta data directly.